src/valkey-benchmark.c

```diff
@@ -1235,17 +1179,12 @@ static int fetchClusterSlotsConfiguration(client c) {
     if (is_fetching_slots) return -1; // TODO: use other codes || errno ?
     atomic_store_explicit(&config.is_fetching_slots, 1, memory_order_relaxed);
     fprintf(stderr, "WARNING: Cluster slots configuration changed, fetching new one...\n");
+    fprintf(stderr, "If you are using the --rfr option and sending write requests (set type commands),\nthe requests could not be processed properly.\n");
+
```

Comment:
I would suggest avoid make this printed for each error. I would think it might be nivce to have it printed once in case the --rfr option is used with something other than pthe default.

```suggestion
```

Reply:
I think It's unnecessary now. When I put this line at first, the description or operation of the options was not clear. So  It was necessary at that time, but it's unnecessary now because everything is clear.